features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/des/des_alt.c

```diff
@@ -14,33 +14,32 @@
  * limitations under the License.
  */
 
-#if !defined(MBEDTLS_CONFIG_FILE)
-#include "mbedtls/config.h"
-#else
-#include MBEDTLS_CONFIG_FILE
-#endif
+#include "mbedtls/des.h"
 
 #if defined(MBEDTLS_DES_C)
 #if defined(MBEDTLS_DES_ALT)
 
 #include <string.h>
-#include "mbedtls/des.h"
-#include "des_alt.h"
+#include <stdbool.h>
 #include "crypto-misc.h"
 #include "nu_bitutil.h"
 #include "mbed_toolchain.h"
+#include "mbed_error.h"
 
-// Must be a multiple of 64-bit block size
+/* DMA buffer
+ * 
+ * MAXSIZE_DMABUF must be a multiple of 64-bit block size.
+ * Its value is estimated to trade memory footprint off against performance. 
```

Comment:
Same comment as with the AES buffer, we should expand the documentation here to say clearly the other constraints on the DMA buffer.

Reply:
@andresag01 Same as above. I refine comment in https://github.com/ARMmbed/mbed-os/pull/4925/commits/b6d4371041e165f9efb19085487eec6369b73a8e.